code/API_definitions/Application_Endpoint_Discovery_API.yaml

```diff
@@ -0,0 +1,596 @@
+---
+openapi: 3.0.3
+info:
+  title: Application Endpoint Discovery API
+  version: 0.1.0-wip
+  description: |
+    # Connect to the Nearest Application Endpoint
+    ---
+    # Overview
+    The Application Discovery API extends beyond the capabilities of the
+    Simple Edge Discovery API by not only locating the nearest Edge Cloud Zone
+    but also directly linking to the application endpoints within those zones.
+    This crucial enhancement enables immediate connection to the most relevant
+    service endpoint, optimizing the application's interaction with
+    edge computing resources.
+    # Introduction
+    Operators often host multiple Edge Cloud Zones across various territories.
+    Connecting an application directly to the nearest operational service
+    endpoint minimizes latency because data travels the shortest possible route.
+    Unlike the Simple Edge Discovery API, which only provides the zone location,
+    this API calculates and provides the actual endpoints within the nearest
+    zone. This approach ensures that developers can instantly integrate these
+    endpoints into their applications, improving performance and user
+    experience.
```

Comment:
Not sure if we should keep the word "performance" in "improving performance and user experience"?

Reply:
Agree